<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogSpecialTests</class>
 <widget class="QDialog" name="dialogSpecialTests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Special Tests</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QLineEdit" name="dds3out"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="masclkf"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pdminv">
     <property name="text">
      <string>Change PDM</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="dds3trktxt">
     <property name="text">
      <string>0-32</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="dds1track">
     <property name="text">
      <string>DDS 1 Sweep</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="dds1out"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="lpttest">
     <property name="text">
      <string>LPT Port Test</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QPushButton" name="cftest">
     <property name="text">
      <string>Cavity Filter Test</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="insert">
     <property name="text">
      <string>Sync Test PDM</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QPushButton" name="prevnalin">
     <property name="text">
      <string>Preset Phase Linearity</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="DDS3">
     <property name="text">
      <string>Command DDS 3</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="DDS1">
     <property name="text">
      <string>Command DDS 1</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="dds1txt">
     <property name="text">
      <string>with DDS Clock at</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="dds3track">
     <property name="text">
      <string>DDS 3 Track</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Band</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButton_1G">
        <property name="text">
         <string>1G</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2G">
        <property name="text">
         <string>2G</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3G">
        <property name="text">
         <string>3G</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Video</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButton_VW">
        <property name="text">
         <string>Wide</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_VM">
        <property name="text">
         <string>Mid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_VN">
        <property name="text">
         <string>Narrow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_VX">
        <property name="text">
         <string>XNarrow</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="17" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxTrans">
     <property name="text">
      <string>Trans/Reflect</string>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxFor">
     <property name="text">
      <string>For/Rev</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>RBW</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButton_RBW1">
        <property name="text">
         <string>1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_RBW2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_RBW3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_RBW4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QPushButton" name="pushButtonSetSwitches">
     <property name="text">
      <string>Set Switches</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QPushButton" name="pushButtonSelectFilter">
     <property name="text">
      <string>Select Filter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
